tests/UI/pages/FO/myAccount/orderDetails.js

```diff
@@ -6,9 +6,14 @@ class OrderHistory extends FOBasePage {
     super();
 
     this.pageTitle = 'Order details';
+    this.successMessageText = 'Message successfully sent';
+    this.messageSend = 'Test';
```

Comment:
It should be better added to the test, 
And I think that you can use `campaign/faker/contactUs.js` to create a new message.

```suggestion
```

Reply:
Replace by random value